Web.playground/Pages/HttpPipeline.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+import Html
+import HttpPipeline
```

Comment:
`HttpPipeline` is the framework with the commoncrypto dependency, and so importing this used to break playgrounds. now it all works!